app/src/main/java/org/fossasia/openevent/general/discount/DiscountApi.kt

```diff
@@ -0,0 +1,13 @@
+package org.fossasia.openevent.general.discount
+
+import io.reactivex.Single
+import org.fossasia.openevent.general.auth.DiscountCode
+import retrofit2.http.GET
+import retrofit2.http.Path
+
+interface DiscountApi {
+
+    @GET("events/{event_identifier}/discount-code?include=event")
```

Comment:
Only include event ID, not the whole event

Reply:
id, not event_identifier

Reply:
okay cool!

Reply:
you mean
``` @GET("events/{id}/discount-code?include=id")```

Reply:
See orga app's API